### PR TITLE
[New arch] Adapt empty views depending on file list option

### PR DESCRIPTION
**Pull request by abelgardep**
## Related Issues
App: #3026 

Title and subtitle aligned with oc10
- [x] Available offline icon should be added as svg. It's a png atm.

| All files | Available offline | Shared by link | Uploads | Logs |
| --- | --- | --- | --- | --- |
| empty_files |empty_offline | empty_shared| empty_uploads | empty_logs  |
___

### CODE REVIEW

File: owncloudApp/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -76,6 +76,7 @@
     <string name="prefs_show_hidden_files">Show hidden files</string>
     <string name="prefs_log_open_logs_list_view">Log files</string>
     <string name="prefs_log_no_logs_list_view">Empty log folder!</string>
+    <string name="prefs_log_empty_subtitle">If you enable logging, log files will show up here</string>
```
Comment (JuancaG05) [inline owncloudApp/src/main/res/values/strings.xml:79]:
```suggestion
    <string name="prefs_log_empty_subtitle">Enable logging and log files will show up here</string>
```
I think this way it is more consistent with other strings such as `upload_list_empty_subtitle` 👍

**Review by JuancaG05 — CHANGES_REQUESTED** [review]
Nice visual change @abelgardep 👍! Just some tiny suggestions and we are ready to go

**Review by JuancaG05 — APPROVED** [review]
Ready to QA! 💯

**Comment by jesmrec** [comment]
QA checks

- [ ] File list
- [ ] Av offline
- [ ] Share by link
- [ ] Uploads
- [ ] Logs
- [ ] Share external

**Pull request merged**